tooling-cpu-load/src/main/java/org/jboss/eap/qe/microprofile/tooling/cpu/load/utils/CpuLoadGenerator.java

```diff
@@ -0,0 +1,66 @@
+package org.jboss.eap.qe.microprofile.tooling.cpu.load.utils;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Generates load on CPU. Takes one argument - load duration. Specifies for how long to generate load. Default 1 min.
+ */
+public class CpuLoadGenerator implements Runnable {
+
+    /**
+     * How many threads exhausting CPU will be started
+     */
+    private static final int NUMBER_OF_THREADS = 200;
+
+    /**
+     * Duration of load. Default is 1 min.
+     */
+    private static Duration loadDuration = Duration.ofMinutes(1); // 1 min;
+
+    /**
+     * Reads load duration in millis as parameter
+     */
+    public static void main(String[] args) throws Exception {
+        if (args.length > 0 && args[0] != null && !"".equals(args[0])) {
+            loadDuration = Duration.ofMillis(Long.valueOf(args[0]));
```

Comment:
From the test code this process can be killed by calling `process.destroy()` but it's used just in way of defensive programming. Having `loadDuration` parameter is convenient way how to make sure that load will take just the given time.

Reply:
What if TS process didn't get enough processor time, test is waiting however load ends...?

Reply:
It will still finish after `loadDuration`. But first thing is not to consume all CPU cores if possible. :-) I'll add this thing to javadoc. In my case even I all cpu cores are consumed then test finishes